### ISSUE TITLE
django.utils.translation.ugettext_lazy() is deprecated

### ISSUE THREAD

**Issue by AlexDaniel** [tmm/django-username-email#21]
Currently I'm getting a lot of warnings. The log  is below. Simply changing all occurances of ugettext_lazy to gettext_lazy should do the trick.

```

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/settings.py:5
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/settings.py:5: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    'app_verbose_name': _("Custom User"),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:91
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:91: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    _('email address'),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:94
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:94: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    'unique': _("A user with that email address already exists."),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:97
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:97: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    first_name = models.CharField(_('first name'), max_length=150, blank=True)

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:98
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:98: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    last_name = models.CharField(_('last name'), max_length=150, blank=True)

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:100
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:100: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    _('staff status'),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:102
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:102: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    help_text=_('Designates whether the user can log into this admin site.'),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:105
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:105: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    _('active'),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:107
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:107: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    help_text=_(

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:112
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:112: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    date_joined = models.DateTimeField(_('date joined'), default=timezone.now)

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:120
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:120: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    verbose_name = _('user')

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:121
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:121: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    verbose_name_plural = _('users')

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:157
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:157: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    verbose_name = _('group')

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:158
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/models.py:158: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    verbose_name_plural = _('groups')

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:19
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:19: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    label=_("Email address"),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:26
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:26: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    label=_("Password"),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:38
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:38: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    'invalid_login': _(

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:42
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:42: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    'inactive': _("This account is inactive."),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:104
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:104: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    'password_mismatch': _('The two password fields didn’t match.'),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:112
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:112: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    label=_("Email address"),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:119
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:119: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    label=_("Password"),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:125
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:125: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    label=_("Password confirmation"),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:128
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:128: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    help_text=_("Enter the same password as before, for verification."),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:179
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:179: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    label=_("Email address"),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:186
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:186: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    label=_("Password"),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:187
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/forms.py:187: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    help_text=_(

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/admin.py:17
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/admin.py:17: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    (_('Personal info'), {'fields': ('first_name', 'last_name')}),

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/admin.py:18
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/admin.py:18: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    (_('Permissions'), {'fields': ('is_active', 'is_staff', 'is_superuser',

../../../.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/admin.py:20
  /home/alex/.cache/pypoetry/virtualenvs/foobar-core-6qdzD22P-py3.9/lib/python3.9/site-packages/cuser/admin.py:20: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    (_('Important dates'), {'fields': ('last_login', 'date_joined')}),
```

**Comment by ataylor32** [comment]
Thanks for the report! This was fixed in 1ec719debba8b928a679213e7afe15636c8a8779 and is in versions 2.3.1 (Django 2.2 or 3.0) and 2.4.1 (Django 3.1)

**Comment by AlexDaniel** [comment]
That was quick. Thank you very much!
